user/conditions-v1.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Conditions
+layout: en
+---
+
+> This page documents the behavior of conditions `v1`, which is currently
+> available using `conditions: v1` in your `.travis.yml` file. This new
+> version replaces `v0` which is currently the default version, and in the
+> process of being deprecated in favor of `v1`. The documentation for `v0` can
+> be found [here](/user/conditions-v0).
+
+Conditions can be used to filter out, and reject builds, stages, and jobs by
+specifying conditions in your build configuration (your `.travis.yml` file).
+See [Conditional Builds, Stages, and Jobs](/user/conditional-builds-stages-jobs)
+for details.
+
+## Examples
+
+This condition requires the branch name to be master (note for PRs this is the
```

Comment:
This one is also really hard to parse. 

I think it would be useful to start off earlier explaining that by including a condition we're automtically exlcuding others? Ie, with branches. (not related to v0 and v1)

Reply:
Maybe it would be more clear to describe the condition as 

> Build only when on the master branch.

Reply:
I agree with @plaindocs I think this wording works well:
> ## Examples
> 
> This condition defines a job which only builds for the master branch.